projectile.el

```diff
@@ -2640,6 +2640,41 @@ is chosen."
 
 (projectile-commander-bindings)
 
+(defun projectile-skel-read-variable ()
```

Comment:
Because it is used in the skeleton below, but it can be used generally, so i should probably rename it.